src/sentry/static/sentry/app/views/settings/organizationRelays/relays.tsx

```diff
@@ -34,154 +38,237 @@ type State = AsyncComponent['state'] & {
   relays: Array<Relay>;
 };
 
-const relaysMock: Array<Relay> = [
-  {
-    public_key: '1:bb:6e:af:66:b4:38:e0:62:83:62:15:22:7',
-    name: 'First key',
-    description: 'optional description for the key',
-    first_seen: '2020-02-07T15:17:00Z',
-    last_seen: '2020-02-07T15:17:00Z',
-    created: '2020-02-07T15:17:00Z',
-  },
-  {
-    public_key: '2:bb:6e:af:66:b4:38:e0:62:83:62:15:22:7',
-    name: 'Second key',
-    description: 'optional description for the key',
-    first_seen: '2020-02-07T15:17:00Z',
-    last_seen: '2020-02-07T15:17:00Z',
-    last_modified: '2020-02-07T15:17:00Z',
-    created: '2020-02-07T15:17:00Z',
-  },
-];
-
 class Relays extends AsyncComponent<Props, State> {
+  constructor(props: Props, context: AsyncComponent['context']) {
+    super(props, context);
+    this.setRelays = this.setRelays.bind(this);
+    this.handleDelete = this.handleDelete.bind(this);
+    this.successfullySaved = this.successfullySaved.bind(this);
+    this.handleOpenEditDialog = this.handleOpenEditDialog.bind(this);
+    this.handleOpenAddDialog = this.handleOpenAddDialog.bind(this);
+  }
+
   getDefaultState() {
     return {
       ...super.getDefaultState(),
-      relays: relaysMock,
+      relays: [],
+    };
+  }
+
+  getEndpoints(): ReturnType<AsyncComponent['getEndpoints']> {
+    return [['data', `/organizations/${this.props.organization.slug}/`]];
```

Comment:
Do you need to re-fetch the organization? You could use `withOrganization()` to get the current organization data.